cmds/contest/main.go

```diff
@@ -68,7 +68,7 @@ func main() {
 		if l, err := dblocker.New(*flagDBURI, config.LockInitialTimeout, config.LockRefreshTimeout); err == nil {
 			target.SetLocker(l)
 		} else {
-			log.Fatalf("ailed to create locker %q: %v", *flagTargetLocker, err)
+			log.Fatalf("failed to create locker %q: %v", *flagTargetLocker, err)
```

Comment:
Not: the recommended style is to capitalize the first letter of a log message

Reply:
Ehm...there are 5 instances of this in the very same file, I thought the repo was following different conventions! 😮 

I'll fix everything!